### DISCUSSION TITLE
年关走亲访友渡劫助手（已满）

### DISCUSSION THREAD

**Discussion by codefd** [InternLM/Tutorial#416] [project]
### **项目想法**
1. 高情商语料搜集:知乎，小红书等社交平台
2. github相关对话语料搜集:阴阳怪气，答非所问数据集
3. 意图分析，对提问者的问题进行分类，判定询问的领域，如婚恋，收入等，选择不同策略
4. 对象分析，对提问者的身份进行分析，判定关系，选择不同的策略
5. 两套系统，一套用来生成长辈风格的提问，另一套用来分析此时应该怎样回答
6. 其他辅助功能，如图片语音视频的生成
7. 意图分析补充，对问题的意图进行分析（如属于正常询问或是略带挑衅不友善的询问），针对不同的询问语气给出不同的回答
8. 将文字同时生成图片表情包，收集那英、华妃、沈腾等表情包大户的图片，根据情绪匹配生成的文字产出表情包
9. 可以调整输出的情绪发疯程度，按照“正常凡人版-》癫公发疯版”让用户可以自己调节

**Comment by Everfighting** [comment]
大佬tql

**Comment by horacehht** [comment]
大佬tql

**Comment by vansin** [comment]
大佬 tql
